### DISCUSSION TITLE
lefigaro.fr: nuisace

### DISCUSSION THREAD

**Discussion by Pierre7855** [uBlockOrigin/uAssets#25030] [Q&A]
Hello,
On this page : https://www.lefigaro.fr/

I am trying uBo Lite.

At the center of the page we have autoplay videos. 
But my main problem is that when I scroll down the page, these autoplay videos are displaying in overlay of the page, at the bottom right. And I am unable to remove them (videos at the bottom right).

**Comment by gorhill** [comment]
Need `lefigaro.fr##.figaro-player.is-sticky` in one of the annoyances list.

**Reply by garry-ut99** [reply]
I've reported here: 
- https://github.com/easylist/easylist/issues/18415#issuecomment-2308502218

Besides / by the way, an information: it's already in _AdGuard – Annoyances_ ==> _AdGuard – Popup Overlays_ (in uBO)
(not that I'm saying it shouldn't be added also to _EasyList Annoyances_ (that way it will be copied to uBOL), I'm just providing an info).

<hr>

By the way, the video doesn't load with _EasyList – Cookie Notices_ on both uBO & uBOL (no problem on _AdGuard – Cookie Notices_), I've reported here:
- https://github.com/easylist/easylist/issues/16487#issuecomment-2308495508

**Reply by Yuki2718** [reply]
Fixed in Fanboy/EasyList Annoyances too https://github.com/easylist/easylist/commit/e76c4497461015b2551194124cc3980755b8e76a
